LocalPackages/DataBrokerProtection/Sources/DataBrokerProtection/UI/UIMapper.swift

```diff
@@ -22,36 +22,6 @@ import os.log
 
 struct MapperToUI {
 
-    func mapToUI(_ dataBroker: DataBroker, extractedProfile: ExtractedProfile) -> DBPUIDataBrokerProfileMatch {
```

Comment:
All this has now moved into constructors on the individual UI objects

Reply:
Nice one!